src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala

```diff
@@ -158,21 +161,44 @@ trait DataFrameSuiteBaseLike extends SparkContextProvider
   }
 
   /**
-   * Zip RDD's with precise indexes. This is used so we can join two DataFrame's
-   * Rows together regardless of if the source is different but still compare
-   * based on the order.
-   */
+    * Zip RDD's with precise indexes. This is used so we can join two DataFrame's
+    * Rows together regardless of if the source is different but still compare
+    * based on the order.
+    */
   private[testing] def zipWithIndex[U](rdd: RDD[U]) = {
-    rdd.zipWithIndex().map{ case (row, idx) => (idx, row) }
+    rdd.zipWithIndex().map { case (row, idx) => (idx, row) }
   }
 
   def approxEquals(r1: Row, r2: Row, tol: Double): Boolean = {
     DataFrameSuiteBase.approxEquals(r1, r2, tol)
   }
+
+  def schemaEquals(expected: DataFrame, result: DataFrame): Unit = {
```

Comment:
For consistency(ish), I would call this `assertSchemaEquals` the approximate style is that if a function is going to assert inside of its self and return a unit we prefix it with `assert` but if a function is going to return a bool that the user will need to assert on we don't have the assert. (I'm open to changing this part of the style if people don't like it).